src/Graph.cpp

```diff
@@ -136,51 +136,52 @@ void Graph::serializeMap()
     }
 }
 
-Graph Graph::loadGraphFromBinary(string graphName, string lockFile, bool nodesHaveTypes, bool lockedSameName)
+void Graph::loadGraphFromBinary(Graph& g, string graphName, string lockFile, bool nodesHaveTypes, bool lockedSameName)
```

Comment:
This should help a lot , a full copy of graph :)

Reply:
Yeah, it'll save 2x memory off this alone I think...